### PR TITLE
TENT-2632 Add accordion table #minor

### PR DESCRIPTION
**Pull request by cfree**
The [TENT-2487](https://jira.sendgrid.net/browse/TENT-2487) story adds a "Templates" accordion [from the style guide](http://styleguide.sendgrid.com/accordions.html) to the transactional templates index page. This PR adds a new generic `<TableAccordion>` and related `<TableAccordionPanel>` component.

## Merge Checklist

- [x] Includes link to associated Github Issue or JIRA ticket
- [x] Has descriptive PR title and body
- [x] All changes are backwards compatible
- [x] QA'd by the SendGrid Style Guide designer
- [ ] Code reviewed by a SendGrid frontend lead
- [x] If adding a new component, the component has been added to the exports in `src/index.ts`.

### CODE REVIEW

File: packages/ui-components/accordion/Accordion.stories.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { storiesOf } from '@storybook/react';
```
Comment (kyliedale) [inline packages/ui-components/accordion/Accordion.stories.js]: normally we don't push up the files from the `/packages` folder. those are compiled from the build

File: src/accordion/accordion.module.scss [hunk 2]
```diff
@@ -176,3 +223,56 @@
...
+      width: 60%;
+      height: 4px;
+      margin: 14px 0;
+      content: '';
+      background-image: linear-gradient(90deg, #d4dadf, #a9b5c0 50%, #d4dadf),
+        linear-gradient(#d4dadf 4px, transparent 0);
+      background-position: 0 0, 0 0;
+      background-size: 200% 100%, 100% 100%;
+      border-radius: 2px;
+      animation: c 1.5s infinite reverse;
```
Comment (kyliedale) [inline src/accordion/accordion.module.scss]: just curious, what is the `c` for?

File: packages/ui-components/accordion/Accordion.stories.js (hunk 1 above)
Comment (cfree) [reply packages/ui-components/accordion/Accordion.stories.js]: Should `/packages` be added to `.gitignore`?

File: src/accordion/accordion.module.scss (hunk 2 above)
Comment (cfree) [reply src/accordion/accordion.module.scss]: It was the name of an animation that wasn't found in the codebase. Fixed.

File: src/styles/global/animations.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+@import './variables.scss';
+
+@keyframes #{$animations-skeleton-loader} {
```
Comment (cfree) [inline src/styles/global/animations.scss]: CSS Modules + Sass workaround. The name of the keyframe was getting hashed where it was used in `accordion.module.scss`

**Comment by thinkingserious** [comment]
Hello @cfree,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**